tutorials/23_depth_camera_tutorial.md

```diff
@@ -63,7 +63,7 @@ depth camera to a memory reserved in a globally scoped `Image`.
 
 \snippet examples/depth_camera/GlutWindow.cc convert depth to image
 
-The other option is the [ConnectNewRgbPointCloud](https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering6/include/ignition/rendering/DepthCamera.hh#L58) call that receives colored point cloud data on callback.
+The other option is the [ConnectNewRgbPointCloud](https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering7/include/ignition/rendering/DepthCamera.hh#L58) call that receives colored point cloud data on callback.
```

Comment:
this link does not exist yet. I forget what is the preferred approach for branch names in tutorial code links

Reply:
We have a mix right now. It's not too bad to have broken links on `main` because the tutorials aren't published anywhere yet. I don't have a strong preference, but it would be nice to be consistent across all libraries